platform-sdk/docs/proposals/proposal-process.md

```diff
@@ -0,0 +1,119 @@
+# Platform Design Proposal Process
+
+This document describes the process for creating and reviewing platform design proposals.
+
+The status of all active design proposals can be viewed
+at [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) project board.
+
+---
+
+## Design Proposal Process Flow
+
+![](designProposalFlow.svg)
+
+## Creating A Proposal
+
+All proposals have an owner who is responsible for taking it through the process to completion. Proposals must be
+presented in the form of a PR containing the proposal in a subdirectory of `platform-sdk/docs/proposals/`. Proposals may
+contain diagrams and supporting materials. Diagrams must be in SVG format and all content must be in an editable
+format (i.e. no images).
+
+Any source code that was developed as part of the design process may be included in the subdirectory of the proposal
```

Comment:
Disagree on that point. A proposal is a document that should contain everything that belongs to the proposal. All information that is not part of the document is totally irrelevant from my point of view. The only reason why we define a folder for a proposal is to store the images / digramms that are part of the proposal and must be added to the doc.

Reply:
The intent is for proposals that include a public API, it may make sense to develop that API interface and objects and include them as part of the proposal. This was the case for the TSS project. Creating the API was an enormous help in the design process, and including in the proposal makes sense because it already exists and is very specific. There is no need to duplicate work by translating it into a diagram or any other format if it already exists. Another example is a protobuf object, like the `PlatformState.proto` for the Public Merkle API project. I will update the comment to be more clear that it is for APIs, not implementation code.

Reply:
I understand that. My point is more: If you want to add a specific public API to the proposal please inline it to the doc. Based on that you have a document that you can read from start to end.

Reply:
If during design, the interfaces were defined, why create extra work to put it into markdown? How about we leave it up to the proposal owner? I can see if there is a short API, then including it in the doc would make sense. But if it is very long, it would be much easier to be able to jump from file to file.